worker/peergrouper/worker.go

```diff
@@ -45,14 +46,20 @@ type Space interface {
 
 type ControllerNode interface {
 	Id() string
-	Life() state.Life
-	Status() (status.StatusInfo, error)
-	SetStatus(status.StatusInfo) error
 	Refresh() error
 	Watch() state.NotifyWatcher
 	WantsVote() bool
 	HasVote() bool
 	SetHasVote(hasVote bool) error
+}
+
+type ControllerDetails interface {
```

Comment:
So this ends up being the machine fields you need? ControllerDetails  implies something more passive - this has a lifecycle and you can watch it. What about ControllerHost?

Reply:
ok